uberfire-security/uberfire-security-management/uberfire-security-management-wildfly/uberfire-security-management-wildfly10/src/test/resources/logback-test.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@psiroky yeahh just saw the debug messages you're right sorry! are good np :)
@manstis hehe np! It's good having you're eyes always looking everywhere :)